### PR TITLE
Update instructions for making a release

### PR DESCRIPTION
**Pull request by bouweandela**
Update the instructions for making a release

### CODE REVIEW

File: doc/contributing.rst [hunk 1]
```diff
@@ -253,42 +253,98 @@ How to make a release
...
+``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files.
+Also update the release date in ``CITATION.cff``.
+See https://semver.org for more information on choosing a version number.
+Make a pull request and get it merged into ``master``.
+
+3. Add release notes
+~~~~~~~~~~~~~~~~~~~~
+Use the script
+`esmvaltool/utils/draft_release_notes.py <https://docs.esmvaltool.org/en/latest/utils.html#draft-release-notes-py>`__
+to create create a draft of the release notes.
```
Comment (valeriupredoi) [inline doc/contributing.rst:278]: probably best to add that we need `pygithub` and can be installed by running `pip install pygithub` and a personal token obtainable by following instructions at https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line (I know the script is verbose and tells you that, but let's have the release manager have the tools installed already and not see it fail the first time)

File: doc/contributing.rst [hunk 2]
```diff
@@ -253,42 +253,98 @@ How to make a release
...
+This script uses the titles and labels of merged pull requests since the
+previous release.
+Review the results, and if anything needs changing, change it on GitHub and
+re-run the script until the changelog looks acceptable.
+Copy the result to the file ``doc/changelog.rst``.
+Make a pull request and get it merged into ``master``.
```
Comment (valeriupredoi) [inline doc/contributing.rst:284]: worth saying they should add at least two of the Core Team as PR reviewers

File: doc/contributing.rst [hunk 3]
```diff
@@ -253,42 +253,98 @@ How to make a release
...
+
+3. Create a release branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create a branch off the ``master`` branch and push it to GitHub.
+Ask someone with administrative permissions to set up branch protection
+rules for it so only you and one other person can push to it.
```
Comment (valeriupredoi) [inline doc/contributing.rst:290]:
```suggestion
rules for it so only you and the admin can push to it.
```

**Review by valeriupredoi — CHANGES_REQUESTED** [review]
couple minor suggestions, dude :+1: :beer:

File: doc/contributing.rst (hunk 1 above)
Comment (bouweandela) [reply doc/contributing.rst:278]: I've improved the documentation here: https://github.com/ESMValGroup/ESMValTool/pull/1848

Comment (valeriupredoi) [reply doc/contributing.rst:278]: yah that's nice, just took a look, good one, man!

File: doc/contributing.rst (hunk 2 above)
Comment (bouweandela) [reply doc/contributing.rst:284]: I should hope that someone making the release knows how to get something merged. However, maybe we should always have the release manager of the previous release as a 'backup release manager', doing it toghether like we are for this release. That would ensure some continuity.

File: doc/contributing.rst (hunk 3 above)
Comment (bouweandela) [reply doc/contributing.rst:290]: That would be a bit vague, who is 'the admin'? shall I write, 'the release manager of the previous release' or 'the backup release manager' instead?

Comment (valeriupredoi) [reply doc/contributing.rst:290]: yeps, sorry, admin is supervague yeah

File: doc/contributing.rst (hunk 2 above)
Comment (bouweandela) [reply doc/contributing.rst:289]: OK, I added this

File: doc/contributing.rst (hunk 3 above)
Comment (bouweandela) [reply doc/contributing.rst]: Added!

**Review by valeriupredoi — APPROVED** [review]
very nice set of instructions @bouweandela :beer:

**Comment by bouweandela** [comment]
Thanks for reviewing!

**Pull request merged**